### PR TITLE
adds chinese workers to your factories

### PR DESCRIPTION
**Pull request by d1rty-pixel**
Just an idea to improve the cookie flow ;)

### CODE REVIEW

File: src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.util.ArrayList;
+
+/**
+ * Created by Niklas on 09.07.2017.
+ *
+ */
+public class ChineseWorkers extends Upgrade{
+
+    public SturdierConveyorBelts(Game game) {
```
Comment (NiklasEi) [inline src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java:20]: We need the ChineseWorkers constructor here

File: src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        super(game, 13);
```
Comment (NiklasEi) [inline src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java:21]: And the ID has to be unique. Simplest would be to use 1000. I am using the official IDs from the original game and they go up to a few hundred.

Comment (NiklasEi) [reply src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java:21]: And the price below at 50 buildings should be at a few billion (50 factory upgrade costs 6.5 billion).

**Comment by NiklasEi** [comment]
It's a nice idea, thank you. The unique ID is nessesary, otherwise the saving and loading won't work.

Maybe we can find a funny quote for the lore:
"New flavor: chicken noodle soup"
Or whatever you can come up with.

File: src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java (hunk 2 above)
Comment (d1rty-pixel) [reply src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java:21]: Sorry about the class constructor, was just a quick idea written down in a simple text editor ;)
Feel free to adapt the costs as you think =)

I will come up with more updates soon ^^

**Comment by TheIntelloBox** [comment]
When is it in the game ?

**Comment by NiklasEi** [comment]
I'll be working on cookie clicker and gamebox in the next days. There will be updates during the next week and I'll include this PR.

**Comment by NiklasEi** [comment]
I'll add all the upgrades up to 250 buildings. You will be able to fully customize the names and lores once i18n is done. Working on that...

**Comment by TheIntelloBox** [comment]
Release date, you think ?

**Comment by NiklasEi** [comment]
I'll upload 0.2 tmr, which will add this upgrade together with many others.

**Comment by TheIntelloBox** [comment]
Fantastic. :)

**Comment by NiklasEi** [comment]
Update is on [Spigot](https://www.spigotmc.org/resources/cookie-clicker-for-gamebox.43801/update?update=179627). The proposed upgrade is avaliable with 25 factories.

**Comment by TheIntelloBox** [comment]
Yes I saw. :)

**Pull request merged**